target/linux/armsr/modules.mk

```diff
@@ -287,6 +287,23 @@ define KernelPackage/octeontx2-net
 endef
 $(eval $(call KernelPackage,octeontx2-net))
 
+define KernelPackage/renesas-net-avb
+SUBMENU:=$(NETWORK_DEVICES_MENU)
+TITLE:=Renesas network drivers
+DEPENDS:=@(TARGET_armsr_armv8) +kmod-phylink +kmod-mii +kmod-ptp +kmod-libphy +kmod-mdio-gpio
+KCONFIG:=CONFIG_NET_VENDOR_RENESAS=y \
+    CONFIG_RAVB \
+    CONFIG_PTP_1588_CLOCK_OPTIONAL=y 
```

Comment:
Selecting `CONFIG_PTP_1588_CLOCK_OPTIONAL` is not needed here.

Reply:
Hi @hauke 
Thanks. I've removed this.